immutablecollections/_immutableset.py

```diff
@@ -286,6 +312,21 @@ def __rsub__(self, other: AbstractSet[V]) -> "AbstractSet[V]":
         """
         return set(item for item in other if item not in self)
 
+    def symmetric_difference(self, other: Iterable[V]) -> "AbstractSet[Union[T, V]]":
+        """
+        Return the union of elements that are in this set but not the other and elements that are
+        in the other set but not this set (`(self - other) | (other - self)`).
+        """
+        if isinstance(other, Set):
+            return self ^ other
+        return self ^ immutableset(other)
+
+    def copy(self) -> "ImmutableSet[T]":
+        """
+        Make a shallow copy of this set.
+        """
+        return copy(self)
```

Comment:
Is this added because `set` and `frozenset` have it?  If so, shouldn't is just be `return self` and documented as such?